pages/api/verify.ts

```diff
@@ -21,45 +22,62 @@ const handler = async (req: NextApiRequest, res: NextApiResponse<Data>) => {
     !body.signature ||
     !body.discordServerId ||
     !body.discordMemberId ||
-    !body.customLink
+    !body.customLink ||
+    !body.network
   ) {
     res.status(400).json({
       message:
-        "Missing body: account, signature, discordServerId, discordMemberId & customLink required",
+        "Missing body: account, signature, DiscordServerId, discordMemberId, customLink & network required",
     });
     return;
   }
 
-  const discordMember = await DiscordMemberRepository.findOne({
+  const discordMembers = await DiscordMemberRepository.find({
     where: {
-      discordServerId: body.discordServerId,
+      DiscordServerId: body.discordServerId,
       discordMemberId: body.discordMemberId,
+      starknetNetwork: body.network,
     },
-    relations: ["discordServer"],
+    relations: ["DiscordServer"],
   });
```

Comment:
why not keep `findOne` since we added `starknetNetwork` to the `where` clause there should be only one match

Reply:
true !